qemu/tests/vm_hugetlb_selftests.py

```diff
@@ -0,0 +1,45 @@
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    vm_hugetlb_selftests test
+    1) Download the current kernel source RPM
+    2) Extract the RPM files
+    3) Extract the linux package and compile the mm selftests
```

Comment:
hi @mcasquer , sorry for replying so late, @YongxueHong and I had a short discussion about your patch, some comments fyi: It might be a little bit heavy to do the first 3 steps, just in order to compile the mm selftest, actually, we can copy the src files into tp-qemu/deps, just like nc_vsock, cpu_flags, then we just copy the src from deps to somewhere, and compile it. Please add more comments if I missed anything @YongxueHong

Reply:
@zhencliu but in that case how do we handle the version? How are the source files updated in the tp-qemu/deps ?

Reply:
Good question, I guessed these selftest src files will not be updated frequently, we can use the latest one. If there is significant change, we can introduce different versions, e.g. deps/iperf.

Reply:
@zhencliu but if this test is supported across different RHEL versions, should we keep different latest RPM versions?

Reply:
hi @mcasquer , if the mm changes frequently, then we need to find another way to install it, @YongxueHong  gave a suggestion that if we can config the installation in ks files(kar/internal_ks dir), prepare the kernel src rpm,  would you please check if this is doable?

Reply:
> hi @mcasquer , if the mm changes frequently, then we need to find another way to install it, @YongxueHong gave a suggestion that if we can config the installation in ks files(kar/internal_ks dir), prepare the kernel src rpm, would you please check if this is doable?

Hi @mcasquer 
yeah, as @zhenyzha mentioned, we suggest preparing the related package before running the case, in other words, can we install it during the installation of the guest so that we do not need to download and then install it for each case?
There is a potential issue with downloading the package, such as the network not working well when downloading the package. In this case, it will block the case from running, but if we install it in advance, not face such a pain point.

Reply:
Thanks @zhencliu @YongxueHong for clearing things up, I will dig into the installation process approach